### DISCUSSION TITLE
'ReactiveProperty<> is an ambiguous reference between 'Reactive.Bindings' and 'ReactiveUI'

### DISCUSSION THREAD

**Discussion by DevEngineReq** [reactiveui/ReactiveUI#3771] [General]
We use viewmodels which inherit from ReactiveObject (ReactiveUI) and which contains ReactiveProperties (Reactive.Bindings)

ReactiveUI -> NugetPackage: ReactiveUI 19.5.72
Reactive.Bindings -> NugetPackage ReactiveProperty 9.5.0

The latest ReactiveUI version also contains a ReactiveProperty. Due to that we get "ambiguous reference" errors.
Unfortunately the new ReactiveProperty in ReactiveUI does not implement ''INotifyDataErrorInfo' and there are no extension methods like 'SetValidateAttribute'. That prevents me from switching the ReactiveUI.ReactiveProperty.

Is it planned to implement INotifyDataErrorInfo?

**Comment by ChrisPulman** [comment]
Hi, thank you for your feedback, I will take a look at what is involved in adding this functionality in the next few days and get back to you.

**Comment by ChrisPulman** [comment]
@DevEngineReq Version 19.6.X will have support for Validation with the ReactiveUI.ReactiveProperty, by using the AddValidation and AddValidationError methods you should be able to achieve the same functionality as Reactive.Bindings.ReactiveProperty
NOTE we will not be adding Xamarin support due to the fact that Xamarin will be out of support at the end of this month.

**Comment by ChrisPulman** [comment]
19.6.1 is now released. I will now close this, please open a new discussion if you have further requests. Many thanks.
